### PR TITLE
Feature/update

### PR DESCRIPTION
**Pull request by kewlfft**
added an update function to upgrade all the AUR (pacaur --update)

### CODE REVIEW

File: aur.py [hunk 1]
```diff
@@ -15,6 +15,18 @@ def package_installed(module, package_name):
...
 
 
+def update_packages(module, tool):
+    assert tool in TOOL_TO_INSTALL_CMD_MAP
+
+    cmd = TOOL_TO_INSTALL_CMD_MAP[tool] + ['--update']
+    rc, stdout, stderr = module.run_command(cmd, check_rc=True)
+
+    module.exit_json(
+        changed='there is nothing to do' not in stdout,
```
Comment (cdown) [inline aur.py]: This is locale dependent, you need to set the `C` locale.

File: aur.py [hunk 2]
```diff
@@ -61,7 +73,7 @@ def main():
     module = AnsibleModule(
         argument_spec={
             'name': {
-                'required': True,
+                'required': False,
```
Comment (cdown) [inline aur.py:76]: This needs checking in `install_packages`, then.

**Review by cdown — CHANGES_REQUESTED** [review]

File: aur.py (hunk 2 above)
Comment (kewlfft) [reply aur.py:76]: thanks for the feedback, changes have been made

**Comment by kewlfft** [comment]
I am pushing again after amendments

**Pull request closed without merging**